client/src/types.ts

```diff
@@ -25,6 +25,13 @@ export interface AskLink {
 }
 export type ChatLink = AskLink | WebLink;
 
+export interface ChatProps {
```

Comment:
I think this type can live just inside the `chat` component? Since it's really just defining what args you can pass to that component